### DISCUSSION TITLE
PoroFullSatTimeDerivative is not a registered object

### DISCUSSION THREAD

**Discussion by ihernandezimbert** [idaholab/moose#29067] [Q&A General]
### Check these boxes if you have followed the posting rules.

- [X] Q&A General is the most appropriate section for my question
- [X] I have consulted the posting Guidelines on the Discussions front page
- [X] I have searched the Discussions forum and my question has not been asked before
- [X] I have searched the MOOSE website and the documentation does not answer my question
- [X] I have formatted my post following the posting guidelines (screenshots as a last resort, triple back quotes around pasted text)

### Question

I am trying to run the borehole_lowres.i file in in the combined modules under poro_mechanics, however I am getting an error that PoroFullSatTimeDerivative is not a registered object. 

I have both solid mechanics and richards loaded in my Makefile, so I am not sure what I am missing.

```
ALL_MODULES         := no

CHEMICAL_REACTIONS  := no
CONTACT             := no
FLUID_PROPERTIES    := yes
HEAT_TRANSFER       := yes
MISC                := no
NAVIER_STOKES       := no
PHASE_FIELD         := yes
RDG                 := no
RICHARDS            := yes
SOLID_MECHANICS     := yes
STOCHASTIC_TOOLS    := no
XFEM                := no
POROUS_FLOW         := yes
```

Any help is appreciated - thank you very much!

**Comment by GiudGiud** [comment]
hello

If you just added the new modules, you need to recompile before they are available.
Unfortunately you often need to also deep clean the app and moose before the modules are built in properly
